tests/test_adaptive_generation.py

```
@@ -0,0 +1,119 @@
+        "findings": []
+    }
+    
+    config = {
+        "adaptive": {
+            "enabled": True,
+            "max_rounds": 2
+        }
+    }
+    
+    with patch("chaos_kitten.brain.orchestrator.AdaptivePayloadGenerator") as MockGen, \
+         patch("chaos_kitten.brain.orchestrator.ChatAnthropic") as MockLLM:
```

Ruff (0.15.1)

[error] Local variable `MockLLM` is assigned to but never used

Remove assignment to unused variable `MockLLM`

(F841)